app.go

```diff
@@ -588,6 +590,11 @@ func (app *App) NewCtxFunc(function func(app *App) CustomCtx) {
 	app.newCtxFunc = function
 }
 
+// RegisterCustomConstraint allows to register custom constraint.
+func (app *App) RegisterCustomConstraint(constraint CustomConstraint) {
```

Comment:
Consider renaming this to `RegisterConstraint()` like the original issue suggested.

Reply:
what about other funcs like `RegisterCustomBinder`?

Reply:
Yeah,  fair if we have functions with that convention to keep it as `RegisterCustomConstraint()`

Reply:
ok, then lets keep it